sheets/README.md

```diff
@@ -0,0 +1,104 @@
+team members, etc.) This had the potential to become a persistent headache for developers, so we decided to support 
+these use cases through UIs that are very familiar to most users regardless of technical expertise: Google Sheets and
+Google Forms. Using various Google Drive and Sheets APIs, the following is now possible: 
+
+1. CS/finance users can make various requests (e.g.: enrollment code creation/assignment, enrollment deferrals) by filling 
+  out a Google Form or entering data into Google Sheets.
+1. Those requests are automatically processed by our app.
+1. Users can monitor the status of their requests directly in the spreadsheets.  
+
+## How It Works
+
+The basic workflow for most flavors of xPro sheets is roughly as follows:
+
+1. A user fills out a Google Form. That form submission is a request for some work to 
+be done, and it is automatically added to a form responses worksheet within a relevant spreadsheet.
+    - That submission also appears in the main worksheet within the spreadsheet automatically via a query.
+1. In response to the change in the spreadsheet, Google sends a file watch request to an endpoint in our app, which 
+  indicates to us that the spreadsheet has been changed, so we may have some new requests to handle.
+1. Our app scans the main worksheet of the spreadsheet for new or updated rows and takes the appropriate action if the
+  data is all valid (e.g.: issuing a refund, creating new enrollment codes, etc.)
+1. If a request was completed for some row, or if an error occurred, our app updates certain columns in the 
+  main worksheet to indicate the status of that request.  
+
+## Types of Sheets
+
+Right now there are three sheet types:
+
+1. Enrollment Code Request sheet (a.k.a. Coupon Request sheet)
+1. Enrollment Code Assignment sheets (which are created based on Enrollment Code Request sheet entries)
+1. Change of Enrollment Request sheet. This includes worksheets for the following:
+    1. Refund Requests
+    1. Deferral Requests
+
+#### Enrollment Code Request Sheet & Assignment Sheets
+
+_Details to be filled in later..._
+
+#### Enrollment Change Requests (Refunds, Deferrals)
+
+_Details to be filled in later..._
+
+## Sheets Management
+
+One cardinal rule to follow for all of the "request"-type sheets: **Do not edit the main worksheet directly, unless the 
+columns exist specifically for user input**. Examples of columns that are intended for user input are the Finance 
+columns in the Change of Enrollment sheets.
+
+The terms "main worksheet" and "form responses worksheet" are used below. Here are some
+screenshots for explanation.
+
+**Enrollment Code Request worksheet tabs:**
+![Enrollment Code Request worksheet tabs](images/enrollment-code-worksheet-tabs.png)
+
+**Change of Enrollment Request worksheet tabs:**
+![Change of Enrollment Request worksheet tabs](images/change-of-enrollment-worksheet-tabs.png)
+
```

Comment:
I think this section could use a little into, to make it clear that when there are errors, there are two options for resolution: editing and ignoring. 

After that I think it will be easier to follow.